DigitalLearningSolutions.Data/Models/TutorialContent/TutorialContent.cs

```diff
@@ -0,0 +1,59 @@
+﻿namespace DigitalLearningSolutions.Data.Models.TutorialContent
+{
+    public class TutorialContent
+    {
+        public int Id { get; }
+        public int CandidateID { get; }
+        public int SectionID { get; }
+        public int CustomisationID { get; }
+        public string TutorialName { get; }
+        public string Status { get; }
+        public int TimeSpent { get; }
+        public int AverageTutMins { get; }
+        public int TutScore { get; }
+        public int PossScore { get; }
+        public bool DiagStatus { get; }
+        public int DiagAttempts { get; }
+        public string Objectives { get; }
+        public string VideoPath { get; }
+        public string TutorialPath { get; }
+        public string SupportingMatsPath { get; }

```

Comment:
We don't need to match the names in the old system, so probably worth making some of these names more descriptive? 🙂

Reply:
Yep, I had meant to ask whether I could give them better names! Refactored in 53f828d 🙂